src/Iterators-Tests/ClassifyOnIteratorTest.class.st

```diff
@@ -0,0 +1,25 @@
+"
+A ClassifyOnIteratorTest is a test class for testing the behavior of ClassifyOnIterator
+"
+Class {
+	#name : #ClassifyOnIteratorTest,
+	#superclass : #TestCase,
```

Comment:
You should have a look at IteratorDecoratorTest to profit from generic tests provided. :-)
Any iterator inheriting from IteratorDecorator should have its associated test case inheriting from IteratorDecoratorTest.

Reply:
Sounds like a good thing - I will check that.